.github/workflows/checks.yml

```
@@ -0,0 +1,35 @@
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+jobs:
+  typecheck:
+    runs-on: ubicloud-standard-2
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Build setup
+        uses: ./.github/common/
```

actionlint (1.7.7)

description is required in metadata of "Build common setup" action at "/home/jailuser/git/.github/common/action.yml"

(action)